docs/api.md

```diff
@@ -2579,6 +2580,14 @@ Dispatches a `mousemove` event.
 
 Dispatches a `mouseup` event.
 
+#### mouse.wheel([options])
+- `options` <[Object]>
+  - `deltaX` X delta in CSS pixels for mouse wheel event (default: 0).
+  - `deltaY` Y delta in CSS pixels for mouse wheel event (default: 0).
```

Comment:
Please include an explanation of what positive and negative means in this context.